setup.py

```diff
@@ -19,7 +19,9 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+import setuptools
 import setuptools_scm
-from setuptools import setup
 
-setup(version=setuptools_scm.get_version())
+setuptools.setup(
+    version=setuptools_scm.get_version(write_to="python/lsst/ts/dimm/version.py")
+)
```

Comment:
Note that you also _**need**_ to modify `python/lsst/ts/dimm/__init__.py` in order for this to work. See the Dev Guide for the necessary change.